src/plugin/cursus/Resources/modules/course/components/form.jsx

```diff
@@ -59,14 +59,22 @@ const CourseForm = (props) =>
             options: {long: true},
             help: trans('plain_description_help')
           }, {
-            name: 'meta.duration',
+            name: 'meta.days',
             type: 'number',
-            label: trans('duration'),
-            required: true,
+            label: trans('session_days', {}, 'cursus'),
             options: {
               min: 0,
               unit: trans('days')
             }
+          }, {
+            name: 'meta.hours',
+            type: 'number',
+            label: trans('session_hours', {}, 'cursus'),
+            options: {
+              min: 0,
+              max: 24,
+              unit: trans('hours')
+            }
```

Comment:
You should create a new data type `duration` for this. See `#/main/app/data/types/time` for example (data types are registered in `#/main/app/plugin`).

Reply:
Do you want create a new data type for regroup days and hours ?

Reply:
Yes. We will add options later to choose which fields to display. This will also allow to display the inputs on the same line.

Reply:
The new duration type have to edit two fields "meta.days" and "meta.hours". I can't keep just one value for the duration for the case or the formation should last "0.5 day 3 hours".
I can store the values in one binary number of 64bits and use the bitwise operations but it's not better.